morpheus/_lib/tests/io/test_io.hpp

```diff
@@ -0,0 +1,28 @@
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2022-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include "../test_morpheus.hpp"  // IWYU pragma: associated
+
+namespace morpheus::test {
+
+TEST_CLASS(DataLoaderRegistry);
+TEST_CLASS(Loader);
```

Comment:
Is this file needed?

Reply:
Yes, I think so. It fits in the existing format, and helps to create segregated training blocks in the test output.